pkg/collect/helm.go

```diff
@@ -155,7 +167,11 @@ func getVersionInfo(actionConfig *action.Configuration, releaseName string, coll
 			Values:    values,
 		})
 	}
-	return versionCollect
+	if len(error_list) > 0 {
+		errs, _ := json.MarshalIndent(error_list, "", " ")
+		return nil, errors.New(string(errs))
```

Comment:
Should never happen, but I prefer handling errors whenever possible

```suggestion
		errs, err := json.MarshalIndent(error_list, "", " ")
		if err != nil {
			strs := []string{}
			for _, e := range error_list {
				strs = append(strs, e.Error())
			}
			return nil, errors.New(strings.Join(strs, "\n"))
		}
		return nil, errors.New(string(errs))
```